roqoqo-qryd/tests/integration/api_devices.rs

```diff
@@ -36,12 +34,21 @@ fn test_new_triangular() {
     let apidevice = QRydAPIDevice::from(&device);
     assert_eq!(device.seed(), 1);
     assert_eq!(device.seed(), apidevice.seed());
-    // assert_eq!(device.pcz_theta(), 0.0);
-    // assert_eq!(device.pcz_theta(), apidevice.pcz_theta());
     assert_eq!(device.qrydbackend(), "qryd_emu_cloudcomp_triangle");
     assert_eq!(device.qrydbackend(), apidevice.qrydbackend());
 }
 
+// Test the new function of the tweezer device
+#[test]
+fn test_new_tweezer() {
+    let device = TweezerDevice::new(Some(1), None, None);
+    let apidevice = QRydAPIDevice::from(&device);
+    assert_eq!(device.seed, 1);
+    assert_eq!(device.seed, apidevice.seed());
+    assert_eq!(device.qrydbackend, "qryd_tweezer_device");
+    assert_eq!(device.qrydbackend, apidevice.qrydbackend());
```

Comment:
Similarly to in the source code, for consistency, I would recommend implementing these as functions

Reply:
Ok I'll add them